package.json

```diff
@@ -207,6 +207,7 @@
     "redux-raven-middleware": "1.2.0",
     "redux-thunk": "2.3.0",
     "reselect": "4.0.0",
+    "terser-webpack-plugin": "^4.0.0",
```

Comment:
Do we need to add this requirement since you don't use it directly, no? It should be updated on cozy-script side? Also, this is "just" a devDep ?